.github/workflows/playwright-e2e.yml

```diff
@@ -0,0 +1,62 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+jobs:
+  e2e_test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: |
+          npm install -g yarn
+          yarn install
+
+      # Spawn a zomboienet chain in background
+      # ToDo: switch to zombienet podman when static ports are supported
+      # podman is preferred but currently it is not possible to set static ports
```

Comment:
@hamidra Do we have a GH issue tracking this?

Reply:
Good point.
I have an issue open in zombienet repo https://github.com/paritytech/zombienet/issues/704
I'll open an issue here to track as well.

Reply:
https://github.com/jeluard/swipealot/issues/115